### DISCUSSION TITLE
Synxflow and Hipims

### DISCUSSION THREAD

**Discussion by hcoludfly** [SynxFlow/SynxFlow#38] [Q&A] [answered]
Hello, Professor Xia,I hope you're doing well. I have a question regarding the flood module in Synxflow. Is it equivalent to Hipims? Do both models not consider the rainfall infiltration process? I would greatly appreciate your insights on this.

**Comment by xiaxilin** [comment, marked as answer]
Hello, in the last two years several bug fixes and improvements have been implemented in SynxFlow, so it is different from hipim-cuda now. SynxFlow does support infiltration. See another thread [here](https://github.com/SynxFlow/SynxFlow/discussions/16).
